src/Microsoft.Health.Fhir.Web/DevelopmentIdentityProviderRegistrationExtensions.cs

```diff
@@ -23,6 +23,8 @@ namespace Microsoft.Health.Fhir.Web
 {
     public static class DevelopmentIdentityProviderRegistrationExtensions
     {
+        private const string WRONGAUDIENCECLIENT = "wrongaudienceclient";
```

Comment:
The constant name should be `WrongAudienceClient` to match the other constant casings in the solution.